### ISSUE TITLE
max_decimal_digits is ignored for 0

### ISSUE THREAD

**Issue by MartinKavik** [Torrencem/display_float#1]
Hi @Torrencem.  I think I found a bug:
```rust
let config = FmtFloatConfig::default()
    .max_decimal_digits(2);
assert_eq!(ftoa(0., config), "0.0e-46");   // I would expect "0"

let config = FmtFloatConfig::default()
    .max_decimal_digits(2)
    .force_no_e_notation();
assert_eq!(ftoa(0., config), "0.0000000000000000000000000000000000000000000000");  // I would expect "0"
```
It applies for both `dtoa` and `ftoa`.

Thank you!

**Comment by Torrencem** [comment]
Good catch! It looks like the Ryu backend crate doesn't like being passed 0.0, and in release mode it gives the behavior you write here (it returns a long string of 0's), but in debug mode it panicks. I'm going to add an exception for 0.0 specifically, working on the fix now...

**Comment by Torrencem** [comment]
Issue fixed in latest commit and v0.3 of package. Thanks for the issue!

**Comment by Torrencem** [comment]
(for example:)

```rust
let config = FmtFloatConfig::default()
    .max_decimal_digits(2)
    .add_point_zero(true);

assert_eq!(dtoa(0., config), "0.0");

let config = FmtFloatConfig::default()
    .max_decimal_digits(2)
    .add_point_zero(false);

assert_eq!(dtoa(0., config), "0");
```
